lang/pt_BR.json

```diff
@@ -110,27 +110,27 @@
     "Deleting...": "Excluindo...",
     "Derived from points, number of achievers, and number of players.": "Derivado de pontos, número de conquistas e número de jogadores.",
     "Descending (Z - A)": "Descendente (Z - A)",
-    "Description": "Description",
+    "Description": "Descrição",
     "Details on how the hash is generated for each system can be found <1>here</1>.": "Detalhes sobre como o hash é gerado para cada console podem ser encontrados <1>aqui</1>.",
     "Dev": "Dev",
     "DevQuest": "DevQuest",
     "Developer Compliance": "Conformidade de Desenvolvedor",
-    "Did not trigger": "Did not trigger",
+    "Did not trigger": "Não foi ativado",
     "Disclaimer about ROMs": "Aviso sobre as ROMs",
     "Disclaimers": "Isenções de Responsabilidade",
-    "Do not open tickets for network issues.": "Do not open tickets for network issues.",
-    "Documentation": "Documentation",
-    "Download Emulator": "Download Emulator",
+    "Do not open tickets for network issues.": "Não abra tickets para problemas de rede.",
+    "Documentation": "Documentação",
+    "Download Emulator": "Baixar Emuladores",
     "Download Patch File": "Baixar Arquivo de Patch",
     "Earliest": "Mais antigo",
     "Email addresses must match.": "Os endereços de e-mail devem corresponder.",
     "Email me": "Me envie um e-mail",
     "Email notifications": "Notificações por E-mail",
-    "Emulator": "Emulator",
-    "Emulator Core": "Emulator Core",
-    "Emulator Version": "Emulator Version",
-    "Example: The achievement requires collecting 10 coins. You collected 10 coins, but the achievement didn't unlock.": "Example: The achievement requires collecting 10 coins. You collected 10 coins, but the achievement didn't unlock.",
-    "Example: The achievement requires collecting 10 coins. You collected 5 coins, and it unlocked.": "Example: The achievement requires collecting 10 coins. You collected 5 coins, and it unlocked.",
+    "Emulator": "Emuladores",
+    "Emulator Core": "Núcleo do Emuladoro",
```

Comment:
possible typo: should this be "Emulador"?

Reply:
sorry, it was a writing error, but I already adjusted